src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -23,7 +23,15 @@ trait LibraryApiService extends HttpService with FireCloudRequestBuilding
 
   val libraryServiceConstructor: UserInfo => LibraryService
 
+  val duosAutocompleteUrl = FireCloudConfig.Duos.baseUrl + "/autocomplete?types=disease&q=%s" // does this still work?
```

Comment:
oh sorry forgot to remove that comment, it was just a message to myself because I was confused that the url wasn't working anymore (since it's now dsde-dev instead of dsde-consent-dev)